### ISSUE TITLE
Bug: Signatures Doesn't Interchange

### ISSUE THREAD

**Issue by Locked-Key** [Foundry376/Mailspring#1825]
**Describe the bug**
When selecting an email alias when composing a new email. On the email composer, the email signature of the previous account stacks up on the email signature of the selected email alias.

**Example End Result:**
```
Signature 1
Name: John Doe
Company: ACME Corporation

Signature 2:
Name: John Doe
Company: Speedrunner Corporation
```

**To Reproduce**
1. Click "Compose New Message"
2. Click on "From" email
3. Select Email Alias

**Expected behavior**
The email alias's signature should replace the current email account's signature on the email body when switching.

**Setup: (please complete the following information):**
 - OS: Windows x64, Latest Stable
 - Mailspring Version: 1.7.4-d54ff730

**Comment by bengotow** [comment]
Hey folks! This should be fixed in the next update (1.7.5) stay tuned!

**Comment by bengotow** [comment]
Hey folks, thanks for reporting this. Happy to say this has been resolved in 1.7.6 which shipped this afternoon. 🚀  The app should auto-update, but if it doesn't you can also grab the new release from GitHub's Releases tab or from https://getmailspring.com/download. Thanks for your patience and help tracking this down, enjoy the update!
